enterprise_gateway/services/processproxies/processproxy.py

```diff
@@ -56,6 +56,10 @@
 # Pop certain env variables that don't need to be logged, e.g. remote_pwd
 env_pop_list = ["EG_REMOTE_PWD", "LS_COLORS"]
 
+# Comma separated list of env variables that shouldn't be logged
+sensitive_env_keys = os.getenv("EG_SENSITIVE_ENV_KEYS", "").split(",")
```

Comment:
We should probably ensure these are lowercased prior to comparison.
```suggestion
sensitive_env_keys = os.getenv("EG_SENSITIVE_ENV_KEYS", "").lower().split(",")
```

Reply:
Done!